src/main/java/life/genny/qwandaq/models/GennySettings.java

```diff
@@ -90,4 +89,29 @@ public static String keycloakUrl() {
 			: "http://keycloak.genny.life";
 	}
 
+	//MINIO
+	public static String getMinIOServerUrl(){
+		return System.getenv("MINIO_SERVER_URL") != null
```

Comment:
Extract to "Constants". Also, store value of System.getenv() into a variable, and return that variable if not null. That would avoid 2 function calls.

Reply:
/**
	 * NOTE: The variables above seem to be defaulting all the time.
	 * likely due to an issue with the environment variables not being present at initialisation.
	 * If it cannot be fixed, we may have to opt for doing it as seen below.
	 **/
Storing the variable to constant gave me issue in runtime so using getters